project.clj

```diff
@@ -30,6 +34,8 @@
   :classifiers [["test" :testutils]]
 
   :profiles {:dev {:dependencies [[spyscope "0.1.4"]
```

Comment:
I use it quite a bit, but agree it doesn't need to be in the project.clj  Should I remove it?